src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,373 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True">
+  <Design.PreviewWith>
+  </Design.PreviewWith>
```

Comment:
What should go within Design.PreviewWith here?

Reply:
Usually, any simple usage example of the control.
Like
```
  <Design.PreviewWith>
        <Border Padding="20">
             <SplitButton />
        </Border>
  </Design.PreviewWith>
```

It will be used by previewer to show some content. We have many of these for Fluent controls styles.

Reply:
Resolved: Recommended PreviewWith details added.